### PR TITLE
Don't show title for empty booking card list

### LINKED ISSUE #313: Do not show empty booking section "PAST" or "UPCOMING"
If there are no relevant bookings, do not show "PAST"/"UPCOMING" header

screen shot 2018-06-01 at 16 41 41

### PR DESCRIPTION
**Pull request by kantord**
fixes #313

Tests are coming up, but feel free to merge it as it is, if you need it for today's release :-) 

Here's how to test it manually: https://skypicker.slack.com/archives/C9K87QN6Q/p1528133034000234?thread_ts=1528132031.000255&cid=C9K87QN6Q<br/><br/><br/><url>LiveURL: https://smartfaq-dont-show-title-for-empty-booking-card-list.surge.sh</url>

### CODE REVIEW

**Comment by jalbertsr** [comment]
I think here there are changes from https://github.com/kiwicom/smart-faq/pull/261, can you rebase plss

File: src/AllBookings/index.js [hunk 1]
```diff
@@ -68,26 +64,8 @@ class AllBooking extends React.Component<Props> {
...
-          <React.Fragment>
-            <div className="subtitle">
-              <Heading weight="medium" size="small">
-                Past
-              </Heading>
-            </div>
-            <BookingCardList booking={past} />
-          </React.Fragment>
-        )}
+        {future && <BookingCardsList booking={future} title="Future" />}
```
Comment (jalbertsr) [inline src/AllBookings/index.js]: the title should be `Upcoming` instead of `Future`

**Comment by kantord** [comment]
Rebased + added tests

**Comment by kantord** [comment]
Rebased. Is it mergable? :-)

**Pull request merged**